src/main/java/com/solvd/shop/models/address/City.java

```diff
@@ -1,9 +1,17 @@
 package com.solvd.shop.models.address;
 
+import jakarta.xml.bind.annotation.*;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "city")
+@XmlType(propOrder = {"cityId", "cityName", "country"})
 public class City {
 
+    @XmlAttribute(name = "id")
     private int cityId;
+    @XmlElement(name = "cityName")
```

Comment:
add empty line between all variables
```suggestion
    private int cityId;
    
    @XmlElement(name = "cityName")
```